game/src/main/kotlin/content/area/misthalin/lumbridge/FredTheFarmer.kt

```diff
@@ -46,6 +47,25 @@ class FredTheFarmer : Script {
                 else -> fullyCompleted()
             }
         }
+        itemOnNPCOperate("ball_of_black_wool", "fred_the_farmer_lumbridge") {
+            when (quest("sheep_shearer_miniquest")) {
+                "started" -> {
+                    giveBlackBallsOfWool > 0
+                    giveBlackWool()
```

Comment:
```suggestion
                    if (giveBlackBallsOfWool > 0) {
                        giveBlackWool()
                    }
```

I presume this is supposed to be